testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -194,6 +194,66 @@ Feature: Be able to manage KVM virtual machines via the GUI
     And I click on "Delete" in "Delete Guest" modal
     Then I should not see a "test-vm2" virtual machine on "kvm_server"
 
+@virthost_kvm
+  Scenario: Refresh a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I click on "Refresh" in tree item "test-pool0"
+    Then I wait until the tree item "test-pool0" has no sub-list
+
+@virthost_kvm
+  Scenario: Stop a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I click on "Stop" in tree item "test-pool0"
+    Then I wait until the tree item "test-pool0" contains "inactive" text
+
+@virthost_kvm
+  Scenario: Start a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I click on "Start" in tree item "test-pool0"
+    Then I wait until the tree item "test-pool0" contains "running" text
+
+@virthost_kvm
+  Scenario: Delete a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I click on "Delete" in tree item "test-pool0"
+    And I check "purge"
+    And I click on "Delete" in "Delete Virtual Storage Pool" modal
+    Then I wait until I do not see "test-pool0" text
+    And file "/var/lib/libvirt/images/test-pool0" should not exist on "kvm_server"
+
+@virthost_kvm
+  Scenario: Create a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I follow "Create Pool"
+    And I wait until I see "General" text
+    And I select "dir" from "type"
+    And I enter "test-pool1" as "name"
+    And I uncheck "autostart"
+    And I enter "/var/lib/libvirt/images/test-pool1" as "target_path"
+    And I enter "0755" as "target_mode"
```

Comment:
what about using a notation a bit more modern ? rwxr-xr-x

:trollface:

Reply:
so far it's passed as-is to libvirt :)